scout/parse/variant/headers.py

```diff
@@ -23,6 +23,42 @@ def parse_rank_results_header(vcf_obj):
     return rank_results_header
 
 
+def parse_local_archive_header(vcf_obj):
+    """Return a dict with local archive data for the case.
+
+    Check if the rank result is in the vcf header. If it exists return a dict
```

Comment:
Isn't this the loqusdb stats and not the rank results?

Reply:
Indeed! 😆